config/formats.js

```diff
@@ -578,7 +578,9 @@ let Formats = [
 				}
 				// @ts-ignore
 				if (pokemon.forte.secondaries) move.secondaries = (move.secondaries || []).concat(pokemon.forte.secondaries);
```

Comment:
I wonder if it would be useful to define a `pokemon.data` object, typed `any`, for use by mods.

So then you wouldn't need to `@ts-ignore` access to it.

Reply:
Would you like me to add it to this pull request?

Reply:
It's already merged, but if you'd like to add it to a different pull request...